.github/workflows/ci.yml

```diff
@@ -41,9 +40,7 @@ jobs:
           node-version: ${{ matrix.node }}
           cache: 'npm'
 
-      - run: npm ci
-      - run: npm run build
-      - run: npm run test --ignore-scripts
+      - run: npm cit
```

Comment:
ohhh, is `cit` a shorthand for `ci` and `test`? TIL

Reply:
correct!